### PR TITLE
Include open source licence information in user guide.

### LINKED ISSUE #3121: Data and code attributions
- [x] data attributions - to list them on the About page 

- [x] code dependencies - currently a link is on the About page; to extend it and presented it more clearly

### PR DESCRIPTION
**Pull request by stevage**
Fixes #3121.

### CODE REVIEW

**Comment by kring** [comment]
Sadly this doesn't work on Windows:
```
$ npm run gulp docs

> terriajs@6.3.7 gulp C:\github\nationalmap\packages\terriajs
> gulp "docs"

[17:04:07] Using gulpfile C:\github\nationalmap\packages\terriajs\gulpfile.js
[17:04:07] Starting 'make-schema'...
[17:04:07] Starting 'oss-attribution'...
[17:04:54] 'oss-attribution' errored after 46 s
[17:04:54] Error: ENOTEMPTY: directory not empty, rmdir 'C:\github\nationalmap\packages\terriajs\node_modules\.bin'
    at Object.fs.rmdirSync (fs.js:846:18)
    at rmkidsSync (C:\github\nationalmap\node_modules\fs-extra\lib\remove\rimraf.js:304:27)
    at rmdirSync (C:\github\nationalmap\node_modules\fs-extra\lib\remove\rimraf.js:281:7)
    at rimrafSync (C:\github\nationalmap\node_modules\fs-extra\lib\remove\rimraf.js:252:7)
    at options.readdirSync.forEach.f (C:\github\nationalmap\node_modules\fs-extra\lib\remove\rimraf.js:291:39)
    at Array.forEach (<anonymous>)
    at rmkidsSync (C:\github\nationalmap\node_modules\fs-extra\lib\remove\rimraf.js:291:26)
    at rmdirSync (C:\github\nationalmap\node_modules\fs-extra\lib\remove\rimraf.js:281:7)
    at rimrafSync (C:\github\nationalmap\node_modules\fs-extra\lib\remove\rimraf.js:252:7)
    at tryCopySync (C:\github\nationalmap\node_modules\fs-extra\lib\move-sync\index.js:99:12)
[17:05:03] Finished 'make-schema' after 56 s
```

The problem is that it's removing the `node_modules/.bin` directory while running something in that directory.

File: gulpfile.js [hunk 1]
```diff
@@ -178,7 +178,34 @@ function runKarma(configFile, done) {
     });
 }
 
-gulp.task('user-guide', ['make-schema'], function() {
+gulp.task('oss-attribution', function() {
+    const fs = require('fs-extra');
+    const path = require('path');
+    let moved = false;
+    // oss-attribution-generator expects (reasonably) that the node_modules directory contains the packages, 
+    // which isn't the case in our usual dev setup
```
Comment (kring) [inline gulpfile.js:186]: Kind of beside the point I guess, but I wouldn't call this reasonable. Node has a sytem for resolving modules, and it's not there just for fun.

**Comment by kring** [comment]
`yarn licenses generate-disclaimer --prod` looks to be a nearly drop-in replacement for `oss-attribution-generator` without any of the problems; 🎉 🕺

**Comment by reginapramesti** [comment]
Went with the `yarn licenses generate-disclaimer` approach in PR #3439

**Pull request closed without merging**